HTML/styles.scss

```diff
@@ -0,0 +1,53 @@
+.level1 {
+    padding: 10px;
+    background-color: #f0f0f0;
+
+    .level2 {
+        padding: 10px;
+        background-color: #d0d0d0;
+
+        .level3 {
+            padding: 10p;
+            background-color: #b0b0b0;
+
+            .level4 {
+                padding: 10px;
+                background-color: #909090;
+
+                .level5 {
+                    padding: 10px;
+                    background-color: #707070;
+                    color: white;
+                }
+            }
+        }
+    }
+}
+
+header nav ul{
+    width: 100%;
+    top: 45px;
+    
+    list-style: none;
+    display: flex;
+    justify-content: space-evenly;
+  
+    }
+  header nav ul li {
+    position: relative;
+  } 
+  header nav ul li:hover {
+    filter: brightness(8);
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Brightness filter value may be excessive</b></div>
<div id="fix">

The `brightness()` filter function typically accepts a percentage or decimal value. The current value of `8` might be too high and could result in an extremely bright or white appearance. Consider using a lower value, such as `1.2` or `120%`, for a subtle brightness increase.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -40,1 +40,1 @@
 -    filter: brightness(8);
 +    filter: brightness(1.2);
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/Amruta101998/DS-and-Algorithm/pull/21#issuecomment-2438027088>#f059f1</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged